src/Microsoft.Data.Sqlite/Interop/NativeMethods`.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Threading;
+using Microsoft.Data.Sqlite.Utilities;
+using static Microsoft.Data.Sqlite.Interop.Constants;
+
+#if NET45 || DNX451 || DNXCORE50
+using Microsoft.Framework.Internal;
+
+#if DNX451 || DNXCORE50
+using System.IO;
+using System.Reflection;
+using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Infrastructure;
+using Microsoft.Framework.DependencyInjection;
+#endif
+
+#endif
+
+namespace Microsoft.Data.Sqlite.Interop
+{
+    internal static partial class NativeMethods
```

Comment:
I split it into one file with pure extern declarations and the other for those that perform some marshaling. The file was beginning to feel unreadable.